ui/design-system-core/src/interfaces/avatar.interface.ts

```diff
@@ -0,0 +1,22 @@
+export type AvatarSrc = { url?: string; fallbackUrl?: string };
```

Comment:
its better if we keep the interface definitions in the file where they are used

Reply:
Alright. I'll fix those